app/addons/documents/doc-editor/components.react.jsx

```diff
@@ -222,7 +245,7 @@ define([
         <div className="panel-section view-attachments-section btn-group">
           <button className="panel-button dropdown-toggle btn" data-bypass="true" data-toggle="dropdown" title="View Attachments"
             id="view-attachments-menu">
-            <i className="icon fonticon-picture"></i>
+            <i className="icon icon-paper-clip"></i>
             <span>View Attachments</span>{' '}
```

Comment:
I'd like to move away from the `{' '}` workarounds and use CSS for all padding instead. More control that way + feels a little less hacky.